fake_spectra/spectra.py

```diff
@@ -665,6 +665,7 @@ def compute_spectra(self,elem, ion, ll, get_tau):
             return result
         for nn in xrange(1,nsegments):
             tresult =  self._interpolate_single_file(nn, elem, ion, ll, get_tau)
+            print("Interpolation %.1f percent done" % (100*nn/nsegments))
```

Comment:
Could you make this only print if it has changed? eg:
if 100*nn % nsegments == 0:
print()

Reply:
Sorry not sure I understand, nn will always be changing because of the loop? And this part of the code isn't parallelised I thought?

Reply:
That's right (none of the python is parallelized). The idea is that instead of printing
"Interpolation 0.1% percent done"
"Interpolation 0.2% percent done"
"Interpolation 0.3% percent done"
"Interpolation 0.4% percent done"
"Interpolation 1.1% percent done"
"Interpolation 1.2% percent done"

etc

you only print
"Interpolation 0% percent done"
"Interpolation 1% percent done"

but thinking it over I changed my mind - having some progress indication is more important.

Reply:
Ok I'll leave it as is! Even in the case of a 1024**3 particle sim, we'd have nsegments=64 so the print output shouldn't be too excessive.

Reply:
Ok, in that case this should be fine!